src/widgets/connector.ts

```diff
@@ -316,6 +396,13 @@ export class ConnectorView extends DOMWidgetView {
         connectionForm.id = "connectionForm";
         connectionFormContainer.appendChild(connectionForm)
 
+        // add a hidden value to hold the alias, this is used when editing a connection
+        const hiddenInput = document.createElement("input");
+        hiddenInput.type = "hidden";
+        hiddenInput.name = "existingConnectionAlias";
+        hiddenInput.value = connectionAlias ? connectionAlias : "";
```

Comment:
We can use the [logical or](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_OR) here, i.e. `connectionAlias || ""`

Reply:
https://github.com/ploomber/jupysql-plugin/pull/70/commits/e48a6b3381d0cdf6900ea629819e7dccc1222c90